.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=c16b9699b21bd898b43cc776da7b018bc4315cb8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Hovering items.chromium.png|0| Removed|

</details>


  <details><summary><b>ContextualMenu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With submenus with hrefs.chromium.png|0| Removed|

</details>


  <details><summary><b>PeoplePicker</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal with text.suggestion- "a".chromium.png|0| Added|
|PeoplePicker.Compact disabled.chromium.png|0| Removed|
|PeoplePicker.List disabled.chromium.png|0| Removed|
|PeoplePicker.Normal disabled with placeholder.chromium.png|0| Removed|
|PeoplePicker.Normal disabled.chromium.png|0| Removed|
|PeoplePicker.Normal with placeholder.chromium.png|0| Removed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|0| Removed|

</details>


  <details><summary><b>TagPicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.TagPicker disabled.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.default.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>